### ISSUE TITLE
EAD3 problem on HTML stylesheet

### ISSUE THREAD

**Issue by elbre** [keeps/roda#1508]
I noticed that similar problem occurred before [#456](https://github.com/keeps/roda/issues/456)

GenericException: Could not process descriptive metadata binary aip/2eae770b-097f-4fde-9137-2eedd86ac7f3/metadata/descriptive/ead3.xml metadata type EAD and version 3, caused by net.sf.saxon.s9api.SaxonApiException: A sequence of more than one item is not allowed as the first argument of fn:normalize-space()

To put it simply, currently html is not able to process two "acqinfo" despite the EAD3 schema is doesn't restricting their number: 

  <xs:complexType name="acqinfo"><xs:sequence>
        <xs:element name="head" type="head" minOccurs="0"/>
        <xs:choice maxOccurs="unbounded">

**Comment by hsilva-keep** [comment]
@elbre have you validated you ead3 xml against the schema? can you provide an example so we can corroborate your claim?

**Comment by elbre** [comment]
Yes, I tried to validate it and the result is valid.
[ead3_censor.txt](https://github.com/keeps/roda/files/3864110/ead3_censor.txt)

**Comment by hsilva-keep** [comment]
@elbre we will assess it as soon as possible and provide feedback...

**Comment by elbre** [comment]
Good day again,
from what I tested in the meanwhile, it seems that this thing is the result of the processing the attributes in general. Basically, if there is attribute used two times and variate only by LABEL, the program ignores the label differentiation and therefore just see one attribute used twice, which had problem to "digest".

It's happening with the "unitid" too despite of the fact that in the schema it usage unlimited.
